qa/util/src/main/java/io/camunda/qa/util/multidb/MultiDbTest.java

```diff
@@ -84,4 +85,15 @@
    *     require OIDC authentication.
    */
   boolean setupKeycloak() default false;
+
+  /**
+   * Setting this is only for local testing purposes; setting the property {@link
+   * CamundaMultiDBExtension#PROP_CAMUNDA_IT_DATABASE_TYPE} will override this. This is NOT a way to
+   * limit which DB a test or class should run with; for that, use JUnit's {@code @DisabledIf}
+   * matching on the property.
+   *
+   * @return the database type to run the test with. By default, the test will run with the local
+   *     database type as defined in the test configuration.
+   */
+  DatabaseType value() default DatabaseType.LOCAL;
```

Comment:
🔧 Not a fan of `value` as a name. How about just calling it `type` or `databaseType`?

Reply:
Ooh hold up, `value` makes it so you don't have to specify the name of the parameter when you want to set it? In that case it does make sense to have it named this way.